packages/utils/src/__tests__/array.test.js

```diff
@@ -3,74 +3,115 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
-import { countDistinct, min, max, toArray } from '../array';
+import { countDistinct, min, max, removeAt, toArray } from '../array';
 
-describe('array', () => {
-  describe('countDistinct', () => {
-    const testData = [
-      ['countDistinct', [[]], 0],
-      ['single item ', [[0]], 1],
-      ['multiple items, same value', [[0]], 1],
-      ['multiple items, different values', [[0, 1]], 2],
-      ['multiple items, mixed same & different values', [[0, 1, 0, 1, 1, 2]], 3],
-      ['function mapper', [[1.1, 2, 1.3], Math.floor], 2],
+describe('countDistinct', () => {
```

Comment:
A lot of whitespace changes here since I un-nested the existing test blocks. For easier reviewing you can review it commit by commit.

The first commit is just un-nesting